lib/helper.js

```diff
@@ -30,16 +31,127 @@ let logofied = text => {
   );
 };
 
+let tablerForArray = data => {
+  if (!Array.isArray(data)) {
+    return "";
+  }
+  let finalTable = new Table({
+    chars: {
+      top: "═",
```

Comment:
Can you extract these configs to a constant instead of duplicating it across both functions?